Ref/Top/RefTopologyAc.cpp

```diff
@@ -0,0 +1,1336 @@
+// ======================================================================
```

Comment:
Yes, these file are auto-generated. However, if we remove them from the repo, then F Prime won't build without (1) an FPP tool installation and (2) a redo installation. Currently neither (1) nor (2) is required. I have been assuming that we will never require (2) and will require (1) only after we have CMake support. Among other things, that keeps CI working while we are in this interim state.

BTW the same thing goes for all the Ai.xml files that are generated from the FPP model and currently checked in to release/v3.0.0. Because they are checked in, the standard F Prime build process (as of 2.x) works on this repo. To remove them, we will need to amend the build process and CI to run the FPP tools.

Reply:
@bocchino got it. I needed to understand that this effects .cpp files as well.  Thanks!